operators/src/error.rs

```diff
@@ -349,6 +353,13 @@ pub enum Error {
         expected: usize,
         found: usize,
     },
+
+    #[cfg(feature = "pro")]
+    QuotaExhausted,
```

Comment:
I guess we said we did not want to have `cfg = pro` all around the code. Could we do something about this?

Reply:
I removed the feature flag. It doesn't hurt to have the enum variant in non-pro and we do it this way all the time, e.g. with permission exceptions in the services crate. 3c7babbdf87ddc47574187df72dcdb57a808070b

Reply:
But now there is a QuotaError also for non-pro, right?

Reply:
Please make a suggestion then.

Reply:
I don't understand this. I always advocate not mixing the pro/non-pro code.

Reply:
> I don't understand this. I always advocate not mixing the pro/non-pro code.

I'm all for that

> This is the only #[cfg(feature = "pro")] that was there.

true. That's why I removed it. I just meant that there are other errors, that are only ever used when a feature is activated (e.g. pro) that are not conditionally compiled.

> What about putting the QuotaError inside the QueryingProcessorFailed or something similar?

that's what I meant here: " I have no other solution here, other than using a generic Box error.". So a new variant `QueryingProcessorFailed { source: Box<dyn Error> }` to hide the real error from pro.

Or do you have a better idea?

Reply:
No, I guess we have to do it this way.

We just have to get sure that we still get a reasonable error out.

Reply:
89751bd532aead2d2f0d99f81480fed7038de92d